.claude/skills/desloppify/SKILL.md

```
@@ -1,226 +1,162 @@
-2. Fix the issue the tool recommends.
-3. `desloppify resolve fixed "<id>"` — or if it's intentional/acceptable:
-   `desloppify resolve wontfix "<id>" --note "reason why"`
-4. Rescan to verify.
+After scanning, **always run `desloppify next`** — it tells you exactly what to do, in order. Don't interpret the scan output yourself or ask the user what to do. Just run `next` and follow its instructions.
 
-**Wontfix is not free.** It lowers the strict score. The gap between lenient and strict IS wontfix debt. Call it out when:
-- Wontfix count is growing — challenge whether past decisions still hold
-- A dimension is stuck 3+ scans — suggest a different approach
-- Auto-fixers exist for open findings — ask why they haven't been run
+The scan will tell you if subjective dimensions need review. Follow its instructions. To trigger a review manually:
+```bash
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)